main/midnight/CMakeLists.txt

```diff
@@ -356,7 +356,11 @@ endif()
 
 if ((NOT APPLE) AND (NOT WINRT))
     ax_get_resource_path(APP_RES_DIR ${APP_NAME})
-    ax_sync_target_res(${APP_NAME} LINK_TO ${APP_RES_DIR} FOLDERS ${content_folder} SYM_LINK 1)
```

Comment:
Windows does not deal well with the attempt to use symlink. Instead, copying both dir contents into 'Content' target directory

Reply:
Only changed for 'WINDOWS' but I am guessing it could make sense also for other target